### DISCUSSION TITLE
如何配置目录

### DISCUSSION THREAD

**Discussion by dongeast** [ppoffice/hexo-theme-icarus#1061] [Q&A]
![image](https://user-images.githubusercontent.com/66114014/162926230-c051cda2-94c6-4362-bae4-1e45065c5fe5.png)
我配置好的博客没有目录模块，我该怎么配置，在配置文件中
![image](https://user-images.githubusercontent.com/66114014/162926355-9b0ac63e-11c9-4962-ace7-3330d6e51a3f.png)
似乎也没有catalogue这个选项

**Comment by dongeast** [comment]
已经知道了，还需要在文章开头标上    toc: true

**Reply by SmallSquare** [reply]
> 已经知道了，还需要在文章开头标上 toc: true

有没有什么办法能默认开启，不想一个一个加😭

**Reply by GeekXtop** [reply]
> > 已经知道了，还需要在文章开头标上 toc: true
> 
> 有没有什么办法能默认开启，不想一个一个加😭

scaffolds下面有个post.md
直接编辑这个模板加上toc: true
之后用hexo new post xxxx新建文章就会自动加上了

**Comment by zzzzsssl** [comment]
我的怎么没有_config.icarus.yml文件呢

**Reply by PJ-568** [reply]
直接运行 hexo g 自动就会生成
